__tests__/app.test.js

```diff
@@ -175,6 +176,64 @@ describe("/api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  it("201: returns the new comment inserted, for current user", () => {
+    const testComment = {
+      username: "mallionaire",
+      body: "lots of social deduction!",
+    };
+
+    const expectedResponse = {
+      author: "mallionaire",
+      comment_added: "lots of social deduction!",
+    };
+
```

Comment:
Hi Nico, returning the comment in this instance requires you return the whole comment object, with all of the properties/columns it has in the database (Like created_at, comment_id etc)
(It should basically be the whole object returned when you use `RETURNING *` at the end of your SQL query)
Can you please update your code and tests to reflect this 🙂